corehq/apps/hqadmin/utils.py

```diff
@@ -0,0 +1,19 @@
+from dimagi.utils.logging import notify_exception
+from pillowtop.utils import get_all_pillows
+
+from .models import PillowCheckpointSeqStore
+
+
+def pillow_seq_store():
+    for pillow in get_all_pillows():
+        checkpoint = pillow.get_checkpoint()
+        store, created = PillowCheckpointSeqStore.objects.get_or_create(checkpoint_id=checkpoint['_id'])
+        if not created and checkpoint['seq'] < store.checkpoint_id:
+            notify_exception(
```

Comment:
fyi, from the cloudant IRC
```
<czue> are cloudant seq IDs string comparable within a single streaming of the changes feed? (https://gist.github.com/smithsz/30fb97662c549061e581 I think implies yes but doesn't say it outright)
<czue> meaning like, if you use since=someseq, will future changes have seq ids that fall alphabetically after the seq you pass in
<czue> (basically asking if the first bit before the hash should always increase)
<rnewson> czue: you should treat update_seq value opaquely, just hand it back as it's given
<rnewson> you should not depend on its 'value' or infer anything from the 'first bit'
```